mne/chpi.py

```diff
@@ -469,14 +463,22 @@ def _setup_hpi_amplitude_fitting(info, t_window, remove_aliased=False,
         raise RuntimeError('Found HPI frequencies %s above the lowpass '
                            '(or Nyquist) frequency %0.1f'
                            % (hpi_freqs[~keepers].tolist(), highest))
-    if info['line_freq'] is not None:
-        line_freqs = np.arange(info['line_freq'], info['sfreq'] / 3.,
-                               info['line_freq'])
-    else:
-        line_freqs = np.zeros([0])
-    logger.info('Line interference frequencies: %s Hz'
-                % ' '.join(['%d' % lf for lf in line_freqs]))
-
+    # calculate optimal window length.
+    if isinstance(t_window, str):
+        if t_window != 'auto':
+            raise ValueError('t_window must be "auto" if a string, got %r'
```

Comment:
To make codecov happy, you can do nowadays:
```suggestion
        _check_option('t_window', t_window, ('auto',), extra='if a string')
```